test/unit/processing/siti-agri/transform-invoice-lines.test.js

```diff
@@ -1,11 +1,11 @@
 const transformInvoiceLine = require('../../../../app/processing/siti-agri/transform-invoice-line')
-const { sfi, sfiPilot, lumpSums } = require('../../../../app/schemes')
+const { sfi, sfiPilot, lumpSums, bps, cs } = require('../../../../app/schemes')
 
 describe('Transform invoice lines', () => {
   test('transforms SFI invoice line', async () => {
     const lineData = ['L', 'SFI0000001', '100', '2022', '80001', 'DRD10', 'SIP00000000001', 'RP00', 'N', '1', 'G00 - Gross value of claim', '2022-12-01', '2022-12-01', 'SOS273']
     const result = transformInvoiceLine(lineData, sfi.schemeId)
-    expect(result).toMatchObject({
+    expect(result).toEqual({
```

Comment:
switched from `toMatchObject` to `toEqual` as it offers deep equality so obejcts have to match on all properties. `toMatchObject` was passing tests even if object was missing a property eg `correlationId`

Reply:
I'm not too sure on this, so I'll let John have a look before resolving this comment

Reply:
Good call, im not 100% on it but it made sense as it made the test a bit stricter but happy to wait and see what John thinks.

Reply:
I'd agree it should be `toEqual` if you want to ensure all properties match.

`toMatchObject` is useful you want to make sure that properties of an object are included along with anything else that may be outside the object.

For really complicated objects you may also need `toStrictEqual` but not in this case.